test/cmd/sender/main.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+	"github.com/google/knative-gcp/pkg/kncloudevents"
+)
+
+const (
+	brokerURLEnvVar = "BROKER_URL"
+)
+
+func main() {
+	brokerURL := os.Getenv(brokerURLEnvVar)
+
+	ceClient, err := kncloudevents.NewDefaultClient(brokerURL)
+	if err != nil {
+		fmt.Printf("Unable to create ceClient: %s ", err)
+	}
+	event := dummyCloudEvent()
```

Comment:
I would just inline the creation here instead of in a function, unless you are using it somewhere else.
Inline will be clearer IMO